modules/openapi-generator/src/main/resources/cpp-pistache-server/model-validation-body.mustache

```diff
@@ -0,0 +1,125 @@
+{{!
```

Comment:
I didn't review this yet, I'll do it once samples have been generated.

Reply:
Samples have been generated.
On that note, running only `./bin/generate-samples.sh` as the PR template states didn't generate the pistache samples, only running
`./bin/generate-samples.sh bin/configs/other/cpp-pistache-server-cpp-pistache.yaml`
seems to have worked.

I'm not sure if this is intentional for the script?

Reply:
Also, for checking the validation code, I recommend running the code through a formatter first, as the indentation is an absolute mess

Reply:
Please move `bin/configs/other/cpp-pistache-server-cpp-pistache.yaml` to `bin/configs/cpp-pistache-server-cpp-pistache.yaml` as part of this PR.

Reply:
> I recommend running the code through a formatter first, as the indentation is an absolute mess

Yup, that's generally what we recommend the users to do after code generation.

Reply:
File moved in ec0bf98